### PR TITLE
feat: create error state

### PR DESCRIPTION
**Pull request by ShakedZrihen**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=WTCP362)

### CODE REVIEW

**Comment by gx0027** [comment]
image
image
image

File: src/@types/svg.d.ts
Comment (ShakedZrihen) [inline src/@types/svg.d.ts:1]: please revert this, no need this change

File: src/components/illustrations/error-filters.svg
Comment (ShakedZrihen) [inline src/components/illustrations/error-filters.svg:1]: this icon is duplicated, you can use same one for both screen.. you can call it `empty-search` and use it both places

File: src/components/illustrations/illustrations.story.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+export default {
+  title: 'Illustrations/IllustrationPage',
+  component: IllustrationPage, IllustrationFilters
+};
+
+export const Error404: StoryObj = {
+  render() {
+    return (
+      <IllustrationPage 
+        message="לא נמצאו פריטים התואמים לחיפוש."
```
Comment (ShakedZrihen) [inline src/components/illustrations/illustrations.story.tsx:17]: please use translation for text

File: src/components/illustrations/illustrationsComponents/IllustrationPage.tsx
Comment (ShakedZrihen) [inline src/components/illustrations/illustrationsComponents/IllustrationPage.tsx:1]: please move it to Error page, not in illustrations..
illustrations should be only for svgs

File: src/components/illustrations/illustrationsComponents/IllustrationSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <Card
+        sx={{
+          padding: 2,
+          textAlign: 'center',
+        }}
+        elevation={2}
+      >
+        <ErrorSpecter />
+        <Typography variant="body2" color="text.primary" sx={{ marginTop: 2 }}>
+          לא נמצאו פריטים התואמים לחיפוש ״מצב הרעב בעזה״. <br />
```
Comment (ShakedZrihen) [inline src/components/illustrations/illustrationsComponents/IllustrationSearch.tsx:22]: please use translation

File: src/components/illustrations/illustrationsComponents/Illustrationfilters.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      <Card
+        sx={{
+          padding: 2,
+          textAlign: 'center',
+        }}
+        elevation={2}
+      >
+        <ErrorSpecter />
+        <Typography variant="body2" color="text.primary" sx={{ marginTop: 2 }}>
+          לא נמצאו פריטים התואמים לפילטור הנבחר.
```
Comment (ShakedZrihen) [inline src/components/illustrations/illustrationsComponents/Illustrationfilters.tsx:17]: please use translation

**Review by ShakedZrihen — COMMENTED** [review]
please move the empty pages to components/custom/Error
and components/custom/EmptyStates

please fix comments

**Pull request closed without merging**